packages/twenty-server/src/modules/messaging/message-import-manager/services/messaging-message.service.ts

```diff
@@ -58,13 +58,14 @@ export class MessagingMessageService {
       });
 
       if (existingMessage) {
-        await messageChannelMessageAssociationRepository.insert(
+        await messageChannelMessageAssociationRepository.upsert(
           {
             messageChannelId,
             messageId: existingMessage.id,
             messageExternalId: message.externalId,
             messageThreadExternalId: message.messageThreadExternalId,
           },
+          ['messageChannelId', 'messageExternalId'],
```

Comment:
@guillim I think this is causing an issue on production as the constraint behind the ON_CONFLICT does not exists

Reply:
fixed in #9851